Benchmarks/AsyncOverhead/AsyncOverhead.fs

```diff
@@ -9,42 +9,84 @@ open System
 open System.Diagnostics
 open System.Threading.Tasks
 
-let doJobAsyncBinds n =
-    let timer = Stopwatch.StartNew ()
-    printf "Job binds: "
-    async {
-      do! Async.SwitchToThreadPool ()
-      for i=1 to n do
-        do ignore i
-        do! Async.Global.ofJob (job { return () })
-    }
-    |> Async.RunSynchronously
-    let d = timer.Elapsed
-    printfn "%d hops in %A" n d
-
-do for n in [100; 1000; 10000; 100000; 1000000; 10000000] do
+let inline (^) x = x
+
+do let doTaskAltBinds n =
+     let timer = Stopwatch.StartNew ()
+     printf "Task-as-Alt-in-Job binds: "
+     let dI = IVar ()
+     job {
+       do! Job.Scheduler.switchToWorker ()
+       for i=1 to n do
+         do ignore i
+         do! dI <|> Alt.fromCancellableTask ^ fun _ -> Task.FromResult ()
+     }
+     |> run
+     let d = timer.Elapsed
+     printfn "%d hops in %A" n d
+
+   for n in [100; 1000; 10000; 100000; 1000000; 10000000] do
+     doTaskAltBinds n
+     GC.Collect ()
+     System.Threading.Thread.Sleep 100
+
+do let doAsyncAltBinds n =
+     let timer = Stopwatch.StartNew ()
+     printf "Async-as-Alt-in-Job binds: "
+     let dI = IVar ()
+     job {
+       do! Job.Scheduler.switchToWorker ()
+       for i=1 to n do
+         do ignore i
+         do! dI <|> Alt.fromAsync ^ async { return () }
+     }
+     |> run
+     let d = timer.Elapsed
+     printfn "%d hops in %A" n d
+
+   for n in [100; 1000; 10000; 100000; 1000000; 10000000] do
+     doAsyncAltBinds n
+     GC.Collect ()
+     System.Threading.Thread.Sleep 100
+
+do let doJobAsyncBinds n =
+     let timer = Stopwatch.StartNew ()
+     printf "Job-in-Async binds: "
+     async {
+       do! Async.SwitchToThreadPool ()
+       for i=1 to n do
+         do ignore i
+         do! Async.Global.ofJob ^ job { return () }
```

Comment:
It probably makes no difference, but is `job { return () }` better than `Job.unit ()`? Same for the `async { return () }` below. I think the computation expressions add an unnecessary `.Delay(…)` call.